### PR TITLE
Add Otel Environment Variables for Tracing configuration

### LINKED ISSUE #3387: Set OTEL endpoint with the OTEL_EXPORTER_OTLP_ENDPOINT env var
**What content needs to be created or modified?**
The associated PR adds the ability to configure the OTEL endpoint for tracing with an environment variable. 

**Describe the solution you'd like**
Add content to the following pages which will describe the environment variables and how they impact the sidecar.

OTEL_EXPORTER_OTLP_ENDPOINT - OpenTelemetry target URL for OTLP exporter
OTEL_EXPORTER_OTLP_INSECURE - OpenTelemetry disables client transport security
OTEL_EXPORTER_OTLP_PROTOCOL - OpenTelemetry transport protocol (grpc, http/protobuf, http/json)

**Where should the new material be placed?**
https://github.com/dapr/docs/blob/v1.10/daprdocs/content/en/operations/configuration/configuration-overview.md
https://github.com/dapr/docs/blob/v1.10/daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**
https://github.com/dapr/dapr/pull/6242

### PR DESCRIPTION
**Pull request by drewby**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [N/A] Commands include options for Linux, MacOS, and Windows within codetabs
- [N/A ] New file and folder names are globally unique
- [N/A ] Page references use shortcodes instead of markdown or URL links
- [N/A ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Adds content to two pages which will describe the environment variables and how they impact the sidecar. Also, corrects a typo in both pages.

OTEL_EXPORTER_OTLP_ENDPOINT - OpenTelemetry target URL for OTLP exporter
OTEL_EXPORTER_OTLP_INSECURE - OpenTelemetry disables client transport security
OTEL_EXPORTER_OTLP_PROTOCOL - OpenTelemetry transport protocol (grpc, http/protobuf, http/json)

## Issue reference
close: #3387

### CODE REVIEW

File: daprdocs/content/en/operations/configuration/configuration-overview.md [hunk 1]
```diff
@@ -79,13 +79,22 @@ The following table lists the properties for tracing:
...
 
 `samplingRate` is used to enable or disable the tracing. To disable the sampling rate ,
 set `samplingRate : "0"` in the configuration. The valid range of samplingRate is between 0 and 1 inclusive. The sampling rate determines whether a trace span should be sampled or not based on value. `samplingRate : "1"` samples all traces. By default, the sampling rate is (0.0001) or 1 in 10,000 traces.
 
+The OpenTelemetry (otel) endpoint can also be configured via an environment variables. The presence of the OTEL_EXPORTER_OTLP_ENDPOINT environment variable
+will turn on tracing for the sidecar.
+
+| Environment Variable | Description |
+|----------------------|-------------|
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | Set the Open Telemetry (OTEL) server address, turns on tracing |
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/configuration/configuration-overview.md]:
```suggestion
| `OTEL_EXPORTER_OTLP_ENDPOINT` | Sets the Open Telemetry (OTEL) server address, turns on tracing |
```

File: daprdocs/content/en/operations/configuration/configuration-overview.md [hunk 2]
```diff
@@ -79,13 +79,22 @@ The following table lists the properties for tracing:
...
+| `OTEL_EXPORTER_OTLP_INSECURE` | Set the connection to the endpoint as unencrypted (true/false) |
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/configuration/configuration-overview.md]:
```suggestion
| `OTEL_EXPORTER_OTLP_INSECURE` | Sets the connection to the endpoint as unencrypted (true/false) |
```

File: daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md [hunk 3]
```diff
@@ -58,3 +58,14 @@ spec:
...
 Changing `samplingRate` to 0 disables tracing altogether.
+
+## Environment variables
+
+The OpenTelemetry (otel) endpoint can also be configured via an environment variables. The presence of the OTEL_EXPORTER_OTLP_ENDPOINT environment variable
+will turn on tracing for the sidecar.
+
+| Environment Variable | Description |
+|----------------------|-------------|
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | Set the Open Telemetry (OTEL) server address, turns on tracing |
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md]:
```suggestion
| `OTEL_EXPORTER_OTLP_ENDPOINT` | Sets the Open Telemetry (OTEL) server address, turns on tracing |
```

File: daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md [hunk 4]
```diff
@@ -58,3 +58,14 @@ spec:
...
+| `OTEL_EXPORTER_OTLP_INSECURE` | Set the connection to the endpoint as unencrypted (true/false) |
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md]:
```suggestion
| `OTEL_EXPORTER_OTLP_INSECURE` | Sets the connection to the endpoint as unencrypted (true/false) |
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
small edits

File: daprdocs/content/en/operations/configuration/configuration-overview.md [hunk 5]
```diff
@@ -79,13 +79,22 @@ The following table lists the properties for tracing:
...
-| `otel.isSecure` | bool | Is the connection to the endpoint address encryped
+| `otel.isSecure` | bool | Is the connection to the endpoint address encrypted
 | `otel.protocol` | string | Set to `http` or `grpc` protocol
 | `zipkin.endpointAddress` | string | Set the Zipkin server address to send traces to
 
 `samplingRate` is used to enable or disable the tracing. To disable the sampling rate ,
 set `samplingRate : "0"` in the configuration. The valid range of samplingRate is between 0 and 1 inclusive. The sampling rate determines whether a trace span should be sampled or not based on value. `samplingRate : "1"` samples all traces. By default, the sampling rate is (0.0001) or 1 in 10,000 traces.
 
+The OpenTelemetry (otel) endpoint can also be configured via an environment variables. The presence of the OTEL_EXPORTER_OTLP_ENDPOINT environment variable
+will turn on tracing for the sidecar.
```
Comment (msfussell) [inline daprdocs/content/en/operations/configuration/configuration-overview.md]:
```suggestion
turns on tracing for the sidecar.
```

File: daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md [hunk 6]
```diff
@@ -58,3 +58,14 @@ spec:
 Dapr uses probabilistic sampling. The sample rate defines the probability a tracing span will be sampled and can have a value between 0 and 1 (inclusive). The default sample rate is 0.0001 (i.e. 1 in 10,000 spans is sampled).
 
 Changing `samplingRate` to 0 disables tracing altogether.
+
+## Environment variables
+
+The OpenTelemetry (otel) endpoint can also be configured via an environment variables. The presence of the OTEL_EXPORTER_OTLP_ENDPOINT environment variable
+will turn on tracing for the sidecar.
```
Comment (msfussell) [inline daprdocs/content/en/operations/monitoring/tracing/setup-tracing.md]:
```suggestion
turns on tracing for the sidecar.
```

File: daprdocs/content/en/operations/configuration/configuration-overview.md [hunk 7]
```diff
@@ -79,13 +79,22 @@ The following table lists the properties for tracing:
...
+
+| Environment Variable | Description |
```
Comment (msfussell) [inline daprdocs/content/en/operations/configuration/configuration-overview.md:92]: Can you also add these environment variables into the overall Env Variable reference topic. This topic https://docs.dapr.io/reference/environment/ in the docs

**Comment by msfussell** [comment]
@drewby - You need to perform a new pull on Dapr v1.11 branch for the build to succeed.

File: daprdocs/content/en/operations/configuration/configuration-overview.md (hunk 7 above)
Comment (drewby) [reply daprdocs/content/en/operations/configuration/configuration-overview.md:92]: Added

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**